play-media-native/features/FormAddConnection/FormAddConnection.tsx

```diff
@@ -96,8 +118,19 @@ export const FormAddConnection = () => {
     setPreviewUrl(text);
   }, []);
 
+  const handleClientID = useCallback((text: string) => {
```

Comment:
Make sure you trim inputs. When pasting, often spaces are following, breaking things.